crates/ruff/src/rules/flake8_pytest_style/rules/raises.rs

```diff
@@ -11,6 +11,35 @@ use crate::registry::Rule;
 
 use super::helpers::is_empty_or_null_string;
 
+/// ## What it does
+/// Checks for `pytest.raises` context managers with multiple statements.
+///
+/// ## Why is this bad?
+/// When a `pytest.raises` is used as a context manager and contains multiple statements, it can
+/// lead to the test passing when it actually should fail. To avoid this, a `pytest.raises`
+/// context manager should only contain a single simple statement that raises the expected
+/// exception.
```

Comment:
I know it's tedious, but do you mind wrapping these to 80 characters? It renders better in the terminal.

Reply:
sure!

Reply:
Thanks :)